packages/amplify-e2e-core/src/init/initProjectHelper.ts

```diff
@@ -46,8 +48,15 @@ export function initJSProjectWithProfile(cwd: string, settings: Object): Promise
 
   addCircleCITags(cwd);
 
+  const cliArgs = ['init'];
+  const providerConfigSpecified = _.isObject(s.providerConfig);
```

Comment:
Instead of relying on lodash, can this be done with `typeof` and a `null` check.